lms/static/js/spec/instructor_dashboard/instructor_dashboard_spec.js

```diff
@@ -0,0 +1,22 @@
+define(['jquery', 'coffee/src/instructor_dashboard/instructor_dashboard'],
+    function($) {
+        'use strict';
+        
+        describe("LMS Instructor Dashboard", function() {
+            
+            beforeEach(function() {
+                loadFixtures('js/fixtures/instructor_dashboard/instructor_dashboard.html');
```

Comment:
This test will be useful for AC-415 so I will move these files there and have them test the functionality of the new tabbed navigation. Thanks!